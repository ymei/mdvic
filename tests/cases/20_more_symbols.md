Inline: $a \leq b \to c$, $A \subseteq B \Rightarrow a \notin C$, $1,2,\ldots,n$, $x \approx y$.

