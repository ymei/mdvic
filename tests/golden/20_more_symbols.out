Inline: a ≤ b → c, A ⊆ B ⇒ a ∉ C, 1,2,…,
n, x ≈ y.
